crt_portal/cts_forms/models.py

```diff
@@ -365,6 +365,9 @@ class ReportAttachment(models.Model):
     report = models.ForeignKey(Report, on_delete=models.CASCADE, related_name='attachments')
     created_date = models.DateTimeField(auto_now_add=True)
 
+    def get_absolute_url(self):
+        return f'attachments/{self.id}'
```

Comment:
We should be using [reverse](https://docs.djangoproject.com/en/2.2/ref/urlresolvers/#reverse) when generating URLs. As-is this is going to fail if/when we want to use it _anywhere_ else besides the report details page.

```suggestion
        return reverse('crt_forms:get-report-attachment', kwargs={"id": self.report.id, "attachment_id": self.id})
```